### DISCUSSION TITLE
Youtube Music not playing

### DISCUSSION THREAD

**Discussion by blambe777** [music-assistant/support#1329] [Q&A]
Hi all
I need some help... I have setup MA and YT music when I try to play something from YT music i get the ding from my speaker and in the MA interface I can see that the track is playing but no sound from my speakers? Everything is working as it should with local music so I am thinking this is an issue with YT music provider. Any ideas what's wrong?

**Comment by OzGav** [comment]
Are you on beta 39 as an issue was resolved in that?

**Comment by BGez** [comment]
I have the same problem, did you found any solution?

**Reply by OzGav** [reply]
Look in the logs when you try and play. Confirm that other providers work fine

**Comment by OzGav** [comment]
Outdated question
